src/calculateRentalCost.js

```diff
@@ -28,7 +28,27 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_FOR_DAY = 40;
+
+  const MIDDLE_TERM = 3;
+  const MIDDLE_TERM_DISCONT = 20;
+
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCONT = 50;
+
+  const RENT_PERIOD = days * RENT_FOR_DAY;
```

Comment:
It is customary to name static variables with a capital letter, if something is calculated in the variable, it is better to name it using the usual camelCase.